normalizer.py

```diff
@@ -2,18 +2,18 @@
 import config
 import sys
 
-def normalizer(data):
-    minHR = float(sys.maxsize)
-    maxHR = float(-1)
+def normalizer(data, keys):
```

Comment:
If we're running this for each column, then don't we only need the single corresponding key? That way instead of doing if data[i] not in keys as you do later, we could just compare it to this single key, which would be more efficient.